### DISCUSSION TITLE
I noticed all mail is getting marked     550error:1C800064:Provider routines::bad decrypt

### DISCUSSION THREAD

**Discussion by graemev** [forwardemail/forwardemail.net#270] [General]
Writing this here because status pages shows 100%  , but I also note all email are failing

**Comment by graemev** [comment]
Since we are in  the land of nodejs (of which I know nothing at all :-) ) ....

https://stackoverflow.com/questions/77890745/why-i-am-getting-the-error-error-error1c800064provider-routinesbad-decrypt

**Comment by titanism** [comment]
This issue is already resolved and was due to a bug with our encryption/decryption helper function 😄
